core/benches/blocks/validate_blocks_oneshot.rs

```diff
@@ -16,7 +18,12 @@ fn main() {
         .expect("Failed building the Runtime");
     {
         let _guard = rt.enter();
-        iroha_logger::test_logger();
+        let config = ConfigReader::new()
+            .with_env(std_env)
+            .read_and_complete()
+            .expect("Failed to load config");
+        let config = InitConfig::new(config, true);
+        let _ = iroha_logger::init_global(config).expect("Failed to initialize logger");
```

Comment:
is this really how we construct config?

Reply:
this is not representative of how clients will do it?

Reply:
You can check how config is constructed in `read_config_and_genesis`.

Here i just wanted to use some default values and override them with env if necessary (not to spam log messages).